odoo/tools/translate.py

```diff
@@ -1158,6 +1164,8 @@ def _babel_extract_terms(self, fname, path, root, extract_method="python", trans
                                  encode(message), comments + extra_comments, value=value)
         except Exception:
             _logger.exception("Failed to extract terms from %s", fabsolutepath)
+            if self._strict_mode:
+                raise
```

Comment:
I guess that you are making this to have the test failling but it looks like it may be better to continue since we may have more errors, no? 

We could catch the logs and make an assertion on them at the end of the test maybe?